hamocc/mo_read_fedep.F90

```diff
@@ -43,7 +43,7 @@ module mo_read_fedep
 
 contains
 
-  subroutine ini_read_fedep(kpie,kpje,omask)
+  subroutine ini_read_fedep(omask)
```

Comment:
I tend to agree with @jmaerz ... *All* HAMOCC routines are taking the indices as arguments, so why change this here?

Reply:
@JorgSchwinger - I'll put the original back in. I was trying to make this consistent with the way streams was being handled.